### PR TITLE
Updating fraud related parameters

### PR DESCRIPTION
**Pull request by tomaszslabon**
Refs #https://github.com/keep-network/tbtc-v2/issues/152
This PR adds functionalities for updating parameters related to frauds in `Bridge`.

### CODE REVIEW

**Comment by pdyraga** [comment]
👀

**Comment by pdyraga** [comment]
👀

File: solidity/test/bridge/Bridge.Frauds.test.ts
Comment (pdyraga) [inline solidity/test/bridge/Bridge.Frauds.test.ts:48]: Non-blocking this PR, especially that this approach is aligned with `updateWalletParameters` function, but I think we should have a separate test file `Bridge.Parameters.test.ts` with tests for `update*` functions. Let's do it at the very end once we have all already opened PRs merged.

**Review by pdyraga — APPROVED** [review]

File: solidity/test/bridge/Bridge.Frauds.test.ts
Comment (tomaszslabon) [reply solidity/test/bridge/Bridge.Frauds.test.ts:48]: Done in the follow-up PR in https://github.com/keep-network/tbtc-v2/pull/229/commits/49f3a5d7a6cabc18a383e4b72c7f4077cdf79499.

**Pull request merged**
